### ISSUE TITLE
In Windows High Contrast Select's list has no border and the chevron is difficult to see

### ISSUE THREAD

**Issue by Paul-Grogan** [carbon-design-system/carbon#6755]
## What package(s) are you using?

- [x] `carbon-components`
- [ ] `carbon-components-react`

## Detailed description

> Describe in detail the issue you're having.

The attached annotated screenshot demonstrates the observations.

With Windows High Contrast enabled, the Selector's opened list has no border, this **may** look to some users that the content of the select are part of the element behind the select (for example contents of a cell in a table).

The Select's chevron is a very dark grey, almost impossible to see against the black background. Should it take on the same colour as the rest of the Select's styling (in my HC config from the screenshot that would be Yellow). Without the chevron being obvious, the select could be mistaken for a regular button.

The Chevron does become visible when choosing either of the two dark grey themes, but that's only because the chevron is a light grey from the theme rather than taking on the HC styling.

> Is this issue related to a specific component?

Select https://www.carbondesignsystem.com/components/select/usage

> What did you expect to happen? What happened instead? What would you like to
> see changed?

Could a visible border be placed on the select list to help put context on the items in the list relative to what's behind.
Could the chevron be styled so that it's visible without having to change the theme?

> What browser are you working in?

Identified on the Edge (Chromium) browser, version 85.0.564.41 (Official build) (64-bit)

> What version of the Carbon Design System are you using?

Assuming the latest because I reproduced it on the Sample pages.

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

WFfH Application Framework, expecting to complete our first ACS certification by the end of September. Which this will impact.

## Steps to reproduce the issue

1. Step one
Enable Windows 10's High Contrast mode

2. Step two
Launch (Edge Chromium) and navigate to the Live Demo section of the Select sample: https://www.carbondesignsystem.com/components/select/usage#live-demo

3. Step three
Examine the Chevron, it will be there, but very difficult to see.
Open the list, there will be no border.

## Additional information

- Screenshots: 
![000669](https://user-images.githubusercontent.com/70535014/91852869-50cbcf00-ec59-11ea-89bb-c83190856b1e.png)

This screenshot is heavily annotated, hopefully it'll be clear enough. One thing to call out is in the top right corner is an example of a select we're using from the Pagination, it opens over a table and initially looks like the elements in the list are part of the cells in the table behind.

**Comment by emyarod** [comment]
related https://github.com/carbon-design-system/carbon/issues/3970

**Comment by dakahn** [comment]
Great catch and just want to say that I absolutely adore annotated screen shots in issues -- great work.

**Comment by Paul-Grogan** [comment]
Just noticed that I never included the example of our table pagination with the select's contents looking like they might be in the cell behind (I referenced it in the description but never added it to the main screenshot).
I'll add that here instead.
![000672](https://user-images.githubusercontent.com/70535014/91974184-538bfa00-ed15-11ea-82b7-3a24f5cbeccc.png)

We were discussing within our team again today, we checked a plain HTML Select from W3Schools, it is behaving just like this in High Contrast too.
https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_select
![000674](https://user-images.githubusercontent.com/70535014/91974365-98b02c00-ed15-11ea-8783-b1cefa140505.png)

This behaviour of the basic Select might impact your direction, either customise by adding the border, or not. 
Regardless of that decision, guess the Chevron will still need to be looked at.

(glad you liked the screenshot :) )

**Comment by pawicao-ibm** [comment]
Hi, recently in my tests I also noticed the mentioned problem with chevron's.
As it's a issue regarding Windows High Contrast mode, I'd like to also add that there's a bug with "X" icons in Modals.
![ModalA11y](https://user-images.githubusercontent.com/67899751/92725468-7ef19300-f36c-11ea-8e6d-ac67c7efa6e2.PNG)
